new-integration-tests/src/test/java/oracle/weblogic/kubernetes/logging/LoggingFacade.java

```diff
@@ -0,0 +1,629 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.logging;
+
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Handler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/** Centralized logging for the operator. */
+public class LoggingFacade {
+
+  public static final String TRACE = "OWLS-KO-TRACE: ";
```

Comment:
can we make this `WKO-TRACE`?

Reply:
This is a copy of the LoggingFacade object in the main line.  You want to have a different value in this version of the facade than the one in mainline?  But, I do like WKO better than OWLS-KO